docs/sources/send-data/otel/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Ingesting logs to Loki using OpenTelemetry Collector
+menuTitle:  Ingesting OpenTelemetry logs to Loki
+description: Ingesting logs to Loki using OpenTelemetry Collector
+aliases: 
+- ../clients/k6/
+weight:  250
+---
+
+# Ingesting logs to Loki using OpenTelemetry Collector
+
+{{% admonition type="warning" %}}
+OpenTelemetry logs ingestions is an experimental feature and is subject to change in future releases of Grafana Loki.
+{{% /admonition %}}
+
+Loki now natively supports ingesting OpenTelemetry logs over HTTP.
+For ingesting logs to Loki using OpenTelemetry Collector, you need to use the [`otlphttp`](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) exporter.
+
+## Loki configuration
+
+When logs are ingested to Loki using OTLP ingestion endpoint, some of the data is stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}).
+Since Structured Metadata is still an experimental feature, Loki by defaults rejects any writes using that feature.
+To start ingesting logs in OpenTelemetry format, you need to enable `allow_structured_metadata` per tenant configuration (in the `limits_config`).
+
+## Configure the OpenTelemetry Collector to write logs into Loki
+
+```yaml
+exporters:
+  otlphttp:
+    endpoint: http://<loki-addr>/otlp
+```
+
+And enable it in `service.pipelines`:
+
+```yaml
+service:
+  pipelines:
+    metrics:
+      receivers: [...]
+      processors: [...]
+      exporters: [..., otlphttp]
+```
+
+If you want to authenticate using basic auth, we recommend the [`basicauth`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/basicauthextension) extension:
+
+```yaml
+extensions:
+  basicauth/otlp:
+    client_auth:
+      username: username
+      password: password
+
+exporters:
+  otlphttp:
+    auth:
+      authenticator: basicauth/otlp
+    endpoint: http://<loki-addr>/otlp
+
+service:
+  extensions: [basicauth/otlp]
+  pipelines:
+    metrics:
+      receivers: [...]
+      processors: [...]
+      exporters: [..., otlphttp]
+```
+
+## Format considerations
+
+Since OTLP format differs from the Loki storage model, here is how data in the OTLP format will be mapped to the Loki data model during ingestion:
+
+- Index labels: Resource attributes map well to index labels in Loki, since both usually identify the source of the logs. Because Loki has a limit of 30 index labels, we have selected following resource attributes to be stored as index labels, while the remaining attributes are stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}) with each log entry:
```

Comment:
```suggestion
- Index labels: Resource attributes map well to index labels in Loki, since both usually identify the source of the logs. Because Loki has a limit of 30 index labels, we have selected the following resource attributes to be stored as index labels, while the remaining attributes are stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}) with each log entry:
```